docs/ml-features.md

```diff
@@ -815,7 +815,7 @@ The rescaled value for a feature E is calculated as,
 `\begin{equation}
   Rescaled(e_i) = \frac{e_i - E_{min}}{E_{max} - E_{min}} * (max - min) + min
 \end{equation}`
-For the case `E_{max} == E_{min}`, `Rescaled(e_i) = 0.5 * (max + min)`
+For the case `$E_{max} == E_{min}$`, `$Rescaled(e_i) = 0.5 * (max + min)$`
```

Comment:
Hm, right. Actually we have several instances where latex is used in scaladoc and it doesn't render, like https://spark.apache.org/docs/2.0.0-preview/api/scala/index.html#org.apache.spark.mllib.optimization.LogisticGradient

Most are private classes. It's even in a log message in GradientDescentSuite. This looks OK to me as-is; you're also welcome to clean up all the useless latex in log messages / scaladoc while here.